pkg/registry/clusterrolebinding/policybased/storage.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
> Correct. AuthorizationRuleResolver requires something that can get/list roles/rolebindings.

And it requires that something to bypass any authorization checks so you don't end up in infinite loops, so you can't pass it a normal client.